docs/application/dotnet/guides/nui/resources.md

```diff
@@ -38,8 +67,8 @@ Visual creation automatically determines whether the image is N_PATCH, SVG, or G
 | `CropToMask`         | Flag to determine if main content image should crop to match mask size. |
```

Comment:
Please add "the"

Flag to determine if the main content image should crop to match the mask size.